cms/models.py

```diff
@@ -46,9 +46,9 @@ def get_context(self, request):
         context["programs"] = Program.objects.filter(live=True)
         context["style_src"] = get_bundle_url(request, "style.js")
         context["public_src"] = get_bundle_url(request, "public.js")
-        context["custom_src"] = get_bundle_url(request, "js/home_page/custom.js")
-        context["ajaxchimp_src"] = get_bundle_url(request, "js/home_page/jquery.ajaxchimp.min.js")
-        context["wow_src"] = get_bundle_url(request, "js/home_page/wow.min.js")
+        context["custom_src"] = static("js/home_page/custom.js")
+        context["ajaxchimp_src"] = static("js/home_page/jquery.ajaxchimp.min.js")
+        context["wow_src"] = static("js/home_page/wow.min.js")
```

Comment:
It looks like `custom.js` contains `__webpack_public_path__` as if it were going to be another webpack bundle. I'm not sure what the intent was in the original commit